static/src/javascripts/projects/commercial/commercial-metrics.ts

```diff
@@ -8,7 +8,14 @@ const { browserId } = window.guardian.config.ophan;
 let logged = false;
 const sendMetrics = (): void => {
 	if (logged) return;
-	logged = sendCommercialMetrics(pageViewId, browserId, isDev);
+	window.guardian.adBlockers.onDetect.push((adBlockInUse: boolean) => {
+		logged = sendCommercialMetrics(
+			pageViewId,
+			browserId,
+			isDev,
+			adBlockInUse,
+		);
+	});
```

Comment:
I don’t like the fact that we would not send commercial metrics if for any reasons `window.guardian.adBlockers.onDetect` fails to trigger its callbacks.

It’s rather initialise `let adBlockInUse: boolean`, and if it’s available when we send metrics we send it. Otherwise we pass `undefined`.